odoo-openmrs/src/test/java/com/ozonehis/eip/odoo/openmrs/mapper/odoo/PartnerMapperTest.java

```diff
@@ -126,6 +128,7 @@ private static Patient getPatient() {
         Patient patient = new Patient();
         patient.setId("123");
         patient.setActive(true);
+        patient.setBirthDate(new Date(883414538000L));
```

Comment:
I don't like setting the date with a long value, as it's not easy to read. Maybe try using `Calendar` or `SimpleDateFormat`?